go.sum

```diff
@@ -104,6 +104,8 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/dnephin/pflag v1.0.7 h1:oxONGlWxhmUct0YzKTgrpQv9AUA1wtPBn7zuSjJqptk=
 github.com/dnephin/pflag v1.0.7/go.mod h1:uxE91IoWURlOiTUIA8Mq5ZZkAv3dPUfZNaT80Zm7OQE=
+github.com/docker/go-units v0.5.0 h1:69rxXcBk27SvSaaxTtLh/8llcHD8vYHT7WSdRZ/jvr4=
+github.com/docker/go-units v0.5.0/go.mod h1:fgPhTUdO+D/Jk86RDLlptpiXQzgHJF7gydDDbaIK4Dk=
```

Comment:
Odd.  I thought we this module (`elastic-agent-shipper`) was already using `github.com/docker/go-units` as an indirect dependency via the `github.com/elastic/beats/v7` direct dependency:

```
 ~/development/github/elastic-agent-shipper   main  go mod graph | grep go-units
github.com/elastic/beats/v7@v7.0.0-alpha2.0.20220810153818-dd118efed5a5 github.com/docker/go-units@v0.4.0
```

Wonder why there was no mention of `github.com/docker/go-units` in `go.sum` already, then? 🤔

Reply:
ah, it's 0.4.0 vs 0.5.0, when I made the dependency explicit it grabbed the latest version

Reply:
But shouldn't there have been two lines for 0.4.0 in `go.sum` already?  I thought `go.sum` captured all dependencies - direct and indirect - but I guess I'm misunderstanding.

Reply:
My bad, it looks like there are entries for `0.4.0` and even `0.3.3` in `go.sum`.  I guess unless you run `go mod tidy` these will stay there.  All good, carry on 🙂.

Reply:
I did run `go mod tidy`; they're still needed as indirect dependencies 😭